### ISSUE TITLE
PDFs preview 

### ISSUE THREAD

**Issue by fabiospampinato** [notable/notable#1018]
Being able to render a preview of PDFs, rather than just linking to them, could be useful.

**Comment by RyanGreenup** [comment]
Yep, this would be amazing, combined with automatically OCR the PDF. I think it would be important to have two options (and a setting to disable/enable the feature with an opt-in not opt-out):

  1. Inline PDF
    * Big/small
  2. Preview in Pane
      * so that you could attach lecture Notes and make notes of them
          * This is done in `org-mode` with [org-noter](https://github.com/weirdNox/org-noter) and [interleaver](https://github.com/rudolfochrist/interleave), I don't use those packages but people tend to love them, the problem is, setting up `org-mode` to work for you, which I have done, is a little like writing a whole bloody operatings system and your config ends up being like 400 lines of LISP and I used a starter kit! ([*Spacemacs*](https://github.com/syl20bnr/spacemacs) for reference)

This would be a kick ass feature that isn't elsewhere implemented.

### OCR

It should however be coupled with an option to OCR and search the PDF (this should definitely also be something that can be toggled).

As it stands I currently take notes on mathematics with GoodNotes 5 on my ipad, but I'll drag and drop textbook snippets, so to make that searchable I'll run it through [OCRMyPDF](https://ocrmypdf.readthedocs.io/en/latest/) and then I can search them with [RipGrep-all](https://github.com/phiresky/ripgrep-all), I'm not sure if those tools could possibly be implemented in someway?

**Comment by fabiospampinato** [comment]
Let's track the OCR part of this here: #995

**Comment by RyanGreenup** [comment]
Ahh I missed that thanks.

**Comment by KraxelHuber** [comment]
Inline PDF Preview would be really helpful!

**Comment by fabiospampinato** [comment]
Closing as duplicate of #368
